tests/test_scene.py

```diff
@@ -260,6 +260,10 @@ def test_scene_upload_sync(dataset):
 
     response = dataset.append(scenes, update=update)
 
+    first_scene = dataset.get_scene(scenes[0].reference_id)
+
+    assert first_scene == scenes[0]
+
```

Comment:
Consider adding a simple case to ensure that `!=` also works.

Reply:
Done